### PR TITLE
Added README with custom layout documentation and LICENSE

### PR DESCRIPTION
**Pull request by rodrigopavezi**
@donnapep @stulees please review. cheers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,129 @@
-# widget-rss
-Rise Vision RSS Widget
+# RSS Widget [![Circle CI](https://circleci.com/gh/Rise-Vision/widget-rss.svg?style=svg)](https://circleci.com/gh/Rise-Vision/widget-rss)
+
+## Introduction
+
+The RSS Widget allows you to add an RSS feed directly into your Presentation. You can configure how it will look and also use a custom layout.
+
+
+Project Name works in conjunction with [Rise Vision](http://www.risevision.com), the [digital signage management application](http://rva.risevision.com/) that runs on [Google Cloud](https://cloud.google.com).
```
Comment (stulees) [inline README.md]: @rodrigopavezi Should replace "Project Name" with "RSS Widget"

File: README.md [hunk 2]
```diff
@@ -1,2 +1,129 @@
...
+- [npm](https://www.npmjs.org)
+- [Bower](http://bower.io/)
+- [Gulp](http://gulpjs.com/)
+- [Protractor](http://angular.github.io/protractor/#/), [CasperJS](http://casperjs.org/), [Karma](http://karma-runner.github.io/0.12/index.html), [Mocha](http://mochajs.org/), [Chai](http://chaijs.com/) and [Chai as Promised](https://github.com/domenic/chai-as-promised/) for testing
+
+## Usage
+
+### Custom Layout
+
+For using custom layout you will need to have a template, like the one shown bellow, hosted somewhere accessible through the internet. The RSS content placement on the template is based on the element classes. Thus, you can change the elements as you wish as long as the classes are maintained.
```
Comment (stulees) [inline README.md]: @rodrigopavezi Spelling error "bellow" should be "below"

**Comment by stulees** [comment]
@rodrigopavezi Curious, we already have a LICENSE file for this repo, is it necessary to have the `.md` version too?

File: README.md (hunk 2 above)
Comment (rodrigopavezi) [inline README.md]: @stulees good catch. I will fix. cheers

**Comment by rodrigopavezi** [comment]
@stulees I am just following this repo https://github.com/Rise-Vision/standard-repo-readme-license as I heard a while that we should use it and I believe the advisable is to have it as .md. cheers

**Comment by stulees** [comment]
@rodrigopavezi It's the right license I'm just noticing that we'll end up with two identical licenses in this repo when this PR gets merged. I'm good with your `.md` version, but I would remove the `.txt` version as a part of this PR so we don't end up with two of them, cheers.

**Comment by rodrigopavezi** [comment]
@stulees I removed the text one. Please review. cheers

**Comment by stulees** [comment]
@rodrigopavezi :+1:

**Pull request merged**